react/src/components/ModelCardModal.tsx

```diff
@@ -113,16 +113,15 @@ const ModelCardModal: React.FC<ModelCardModalProps> = ({
             : '90%'
       }
       footer={[
-        // FIXME: ModelTryContentButton is not working properly
-        // It should be fixed in the future.
         // This button is used to clone-and-create/create the model service with the content of the model card.
-        /*<ModelTryContentButton
-          vfolderNode={model_card?.vfolder_node || null}
-          modelStorageHost={model_card?.vfolder?.host as string}
-          modelCardMetadata={model || null}
-          modelName={model_card?.name as string}
-          key="try"
-        />,*/
+        <Suspense key="model-try-content-button">
+          <ModelTryContentButton
```

Comment:
It's better to set fallback like below:
```suggestion
        <Suspense
          key="model-try-content-button"
          fallback={
            <Tooltip title={'설정 확인 중..'}>
              <Button loading disabled />
            </Tooltip>
          }
        >
          <ModelTryContentButton
```